app/src/main/res/values/colors.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#6200EE</color>
```

Comment:
I see you are using colors.xml, that's good - try to keep consistent and keep all colors here